seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlIT.java

```diff
@@ -71,6 +77,9 @@
 import java.util.Map;
 import java.util.Properties;
 
+@DisabledOnContainer(
+        value = {},
+        type = {EngineType.FLINK, EngineType.SPARK})
```

Comment:
Hi @mosence , this is a mistake by me. Could you revert this? Thanks.

Reply:
Yes, I found it.
Why did this happen?

Reply:
I do test it in local. Just want to test faster. So I did this...